externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/operation/SparkOperation.scala

```diff
@@ -98,6 +101,7 @@ abstract class SparkOperation(opType: OperationType, session: Session)
   protected def withLocalProperties[T](f: => T): T = {
     try {
       spark.sparkContext.setJobGroup(statementId, statement, forceCancel)
+      spark.sparkContext.setJobDescription(redactedStatement)
```

Comment:
we can combine this two line by:

`spark.sparkContext.setJobGroup(statementId, redactedStatement, forceCancel)`

Reply:
Thanks. it will be combined. ;-)